.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,33 @@
+name: 💡 Feature Request
+description: Suggest an idea for this project.
+title: "💡 feat: <title>"
+labels: [Enhancement]
+body:
```

Comment:
Maybe here there should also be a checkbox about has checked for possible duplicates

Reply:
I guess we could, but I don't think we get feature requests often enough for this to be an issue. At least, I don't recall ever seeing a duplicate feature request.